commodore/component-template/{{ cookiecutter.slug }}/renovate.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": [
-    "config:base"
+    "config:base",
+    ":gitSignOff",
```

Comment:
I don't think we need Git signed-off-by for components.

 ```suggestion
```

Reply:
Right, but I see no harm either.
Do you prefer it without?

Reply:
I don't really care either way

Reply:
Ok :) Would you mind approving the PR? Merging is blocked with "change requested"